bin/migrate-list

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+// vim: set ft=javascript:
+'use strict'
+
+var program = require('commander')
+var path = require('path')
+var moment = require('moment')
```

Comment:
What about using a light library like `dateformat` rather than `moment` (which is pretty huge)?